src/FixturesKernel.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Util;
+
+
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class FixturesKernel implements HttpKernelInterface
+{
+    public function handle(Request $request, $type = self::MASTER_REQUEST, $catch = true)
+    {
+        $this->prepareSession($request);
+
+        $response = $this->handleFixtureRequest($request);
+
+        $this->saveSession($request, $response);
+        $response->prepare($request);
+
+        return $response;
+    }
+
+    private function handleFixtureRequest(Request $request)
+    {
+
+        $fixturesDir = realpath(__DIR__.'/../web-fixtures');
+        $overwriteDir = realpath(__DIR__.'/../http-kernel-fixtures');
+
+        require_once $fixturesDir . '/utils.php';
+
+        $file = $request->getPathInfo();
+
+        $path = file_exists($overwriteDir.$file) ? $overwriteDir.$file : $fixturesDir.$file;
+
+        $resp = null;
+
+        ob_start();
+        include($path);
```

Comment:
Maybe we should use `require` instead of `include` here to at least get Fatal Error, when in tests somebody tries to access non-existing url.